app/routes/financial.js

```diff
@@ -91,6 +91,21 @@ const FinancialRoute = function(app, db, logger) {
     controller.getData(_key);
   });
 
+  app.put( "/financial/:key", jsonParser, ( req, res, next ) => {
+    if ( !( "value" in req.body ) ) {
+      res.statusCode = 400;
+      next( new Error( "Missing PUT data" ) );
+
+      return;
+    }
+
+    _res = res;
+    _next = next;
+    _key = req.params.key;
+
```

Comment:
@donnapep Thanks.  There's a lot of repetition of the form `_x=x` so it might be nice to have a `bindInstanceVariables()` or something.  I think it would be nice to use `this.x` instead of `_x` too.  It makes it more clear that we're talking about class instance variables and uses the the language features instead of an unenforced convention.